pkg/controller/virtinformers.go

```diff
@@ -543,10 +543,43 @@ func (f *kubeInformerFactory) VirtualMachine() cache.SharedIndexInformer {
 	})
 }
 
+func GetVirtualMachineExportInformerIndexers() cache.Indexers {
+	return cache.Indexers{
+		"pvc": func(obj interface{}) ([]string, error) {
+			export, ok := obj.(*exportv1.VirtualMachineExport)
+			if !ok {
+				return nil, unexpectedObjectError
+			}
+
+			if export.Spec.Source.APIGroup == nil ||
+				*export.Spec.Source.APIGroup == "" || *export.Spec.Source.APIGroup == "v1" &&
```

Comment:
I forgeet the rules of operator precedence, but should `*export.Spec.Source.APIGroup == "" || *export.Spec.Source.APIGroup == "v1"` be in parens?

Reply:
Yeah you are right I think, I also forget but normally put parens regardless to make it clearer what is going on.